server/tokens.go

```diff
@@ -204,27 +294,23 @@ func (c *Server) blindedTokenRedeemHandler(w http.ResponseWriter, r *http.Reques
 				Message: "Could not mark token redemption",
 				Code:    http.StatusInternalServerError,
 			}
-
-		}
-		err := json.NewEncoder(w).Encode(blindedTokenRedeemResponse{verifiedCohort})
-		if err != nil {
-			c.Logger.Error("Could not encode the blinded token")
-			panic(err)
 		}
+		response = blindedTokenRedeemResponse{verifiedCohort}
 	}
-	return nil
+	return handlers.RenderContent(r.Context(), response, w, http.StatusOK)
 }
 
 func (c *Server) blindedTokenBulkRedeemHandler(w http.ResponseWriter, r *http.Request) *handlers.AppError {
-
 	var request BlindedTokenBulkRedeemRequest
 
 	if err := json.NewDecoder(http.MaxBytesReader(w, r.Body, maxRequestSize)).Decode(&request); err != nil {
+		c.Logger.Error("Could not parse the request body")
 		return handlers.WrapError(err, "Could not parse the request body", 400)
 	}
 
 	tx, err := c.db.Begin()
 	if err != nil {
+		c.Logger.Error("Could not start bulk token redemption db transaction")
```

Comment:
should we be logging this

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/c212d77eac67c1cc0369b89c95e4dea5812fd3fe